### DISCUSSION TITLE
Sidebar table of contents not easily discoverable when collapsed

### DISCUSSION THREAD

**Discussion by adamtheturtle** [pradyunsg/furo#124] [Potential Issue] [answered]
I have a handbook and share this handbook with clients.
I screen-share with those clients to get them set up with the handbook and other tools.

A typical task is "Go to X page, now go to the Y section".
I used to use the alabaster theme and an in-line table of contents.
I never encountered a problem with this use case.
People would always (/ usually?) click on the ToC menu item.

I have now switched to Furo (of course I have, it is beautiful!).
Multiple times, I have seen the following problem:

* The client's default window size is not wide enough to show the table of contents in the sidebar
* I say "Now go to the Y section"
* People do not reach for the sidebar "hamburger" menu item, and instead cmd/ctrl+f, which is not ideal.

**Comment by pradyunsg** [comment, marked as answer]
Well, I can't think of any good ways to solve this. :)

Based on what I've seen, most existing documentation themes either collapse directly to a mobile-like view or have a similar issue.

**Reply by adamtheturtle** [reply]
I'm not sure either but I have worked around it by using an inline table of contents at the top.
